app/app/management/commands/sync_orgs_repos.py

```diff
@@ -0,0 +1,189 @@
+                    profile = sync_profile(handle)
+                    print('Profile from sync')
+                    print(profile)
+                    access_token = profile.user.social_auth.filter(provider='github').latest('pk').access_token
+                    print('Removing Stale Organizations and Groups')
+                    remove_org_groups = [x for x in profile.profile_organizations.all() if
+                                         x.name not in profile.organizations]
+                    for y in remove_org_groups:
+                        profile.profile_organizations.remove(y)
+                        profile.user.groups.filter(name__contains=y.name).delete()
+                        print(f'Removing: {profile.handle} from Organization: {y.name} ')
+
+                    user_access_repos = get_repo(handle, '/repos', (handle, access_token), is_user=True)
+                    # Question around user repo acccess if we can't get user repos, should we assume all repos are no longer available in the platform?
+
+                    if 'message' in user_access_repos:
+                        print(user_access_repos['message'])
+                        return lsynced
+
+                    current_user_repos = []
+                    for y in user_access_repos:
+                        current_user_repos.append(y['name'])
+
+                    remove_user_repos_names = [x for x in profile.repos.all() if
+                                               x.name not in current_user_repos]
+
+                    remove_user_repos = Repo.objects.filter(name__in=remove_user_repos_names,
+                                                            profile__handle=handle)
+                    for y in remove_user_repos:
+                        profile.repos.remove(y)
+
+                    lsynced.append(handle)
+                else:
+                    return lsynced
+
+                members_to_sync = []
+                if profile.organizations is None:
+                    print("no organizations to sync")
+                    return []
+
+                for org in profile.organizations:
+                    try:
+                        if org in orgs_synced:
+                            print(f'{org} has been synced already')
+                            continue
+
+                        orgs_synced.append(org)
+                        db_org = Organization.objects.get_or_create(name=org)[0]
+
+                        print(f'Syncing Organization: {db_org.name}')
+                        profile.profile_organizations.add(db_org)
+                        org_members = get_organization(
+                            db_org.name,
+                            '/members',
+                            (handle, access_token)
+                        )
+
+                        if 'message' in org_members:
+                            print(org_members['message'])
+                            continue
```

Comment:
^ do we need these 3 lines

Reply:
I think these are for logging errors in the cronjob log

Reply:
correct we're logging out whats happening, if the returning object has a message key at all from at all its an error, you would want to notify and skip to the next element in the array.